subzero-core/src/main/java/info/jerrinot/subzero/DelegateKryo.java

```diff
@@ -0,0 +1,13 @@
+package info.jerrinot.subzero;
+
+import com.esotericsoftware.kryo.Kryo;
+
+/**
+ * Created by brunomendoncalopes on 11/11/16.
+ *
+ * @author Bruno Lopes
+ */
+public interface DelegateKryo {
```

Comment:
The name of not very descriptive. What about `KryoConfigurer` with `configure()` instead of `accept()`?

also JavaDoc should explain usage of this, with examples if possible